pkg/apis/core/validation/validation.go

```diff
@@ -4036,6 +4039,25 @@ func ValidateNodeUpdate(node, oldNode *core.Node) field.ErrorList {
 	return allErrs
 }
 
+func validateNodeConfigSource(source *core.NodeConfigSource, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	count := int(0)
+	if ref := source.ConfigMapRef; ref != nil {
+		count++
+		// name, namespace, and UID must all be non-empty for ConfigMapRef
+		if ref.Name == "" || ref.Namespace == "" || string(ref.UID) == "" {
```

Comment:
why uid?

Reply:
fine to leave it, I just missed any other use of it in this PR and wondered

Reply:
https://github.com/kubernetes/kubernetes/issues/61643